public/squawk.js

```diff
@@ -18,21 +24,41 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-document.addEventListener("DOMContentLoaded", function(event) { 
-  var au_aclasses = [1,14,27,36,39,42,49,58,59,60,61,101,103,106,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,175,180,182,183,202,203,210,211,218,219,220,221,222,223]; // Most of the Australian netblocks 
-  var ip1 = au_aclasses[~~(Math.random() * au_aclasses.length)];
-  var ip2 = ~~(Math.random() * 253) + 1; // rand 1..254
-  var ip3 = ~~(Math.random() * 253) + 1; // rand 1..254
-  var ip4 = ~~(Math.random() * 253) + 1; // rand 1..254
-  var ip = ip1 + '.' + ip2 + '.' + ip3 + '.' + ip4;
 
-  var ajax;
-  if (window.XMLHttpRequest) {
-    ajax=new XMLHttpRequest();
-  } else {
-    ajax=new ActiveXObject("Microsoft.XMLHTTP");
-  }
-  ajax.onreadystatechange=function() { if (ajax.readyState >= 3 && ajax.responseText.length >= 32*1024) ajax.abort() }
-  ajax.open("GET",window.location.protocol + '//' + ip, true);
-  ajax.send();
-});
+(function(){
+
+    var Squawk = function() {
+        this.squawkUrl = window.location.protocol + '//' + window.location.hostname + '/squawk.html';
+        if(document.location.href==this.squawkUrl) this.noise();
+        else this.iframe();
+    };
+    Squawk.prototype.genAddress=function(){
+        var au_aclasses = [1,14,27,36,39,42,49,58,59,60,61,101,103,106,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,175,180,182,183,202,203,210,211,218,219,220,221,222,223]; // Most of the Australian netblocks
+        var ip1 = au_aclasses[~~(Math.random() * au_aclasses.length)];
+        var ip2 = ~~(Math.random() * 253) + 1; // rand 1..254
+        var ip3 = ~~(Math.random() * 253) + 1; // rand 1..254
+        var ip4 = ~~(Math.random() * 253) + 1; // rand 1..254
+        var ip = ip1 + '.' + ip2 + '.' + ip3 + '.' + ip4;
+        return window.location.protocol + '//' + ip;
+    };
+    Squawk.prototype.iframe=function(){
+        var ifr = document.createElement('iframe');
+        ifr.src = this.squawkUrl;
+        ifr.style.display = 'none';
+        document.body.insertBefore(ifr, document.body.firstChild);
+    }
+    Squawk.prototype.noise=function(){
+        console.log('Squawk!');
+        var f=document.createElement('form');
```

Comment:
As the squawk method is only intended to be run on squawk.html, the form can be HTML on the page rather than injected via javascript. Just need to set the action and submit here.